proto/wg/cosmo/node/v1/node.proto

```diff
@@ -78,6 +78,7 @@ message EngineConfiguration {
   string graphqlSchema = 4;
   repeated TypeConfiguration type_configurations = 5;
   map<string, string> string_storage = 6;
+  optional string graphql_client_schema = 7;
```

Comment:
Out of curiosity. If you remove `optional` what will be the type? It will be still optional correct?

Reply:
without optional
```go
type Some struct {
  ClientSchema string
}
```

with optional
```go
type Some struct {
  ClientSchema *string
}
```

Reply:
I personally like optional better because we can then assume that a non nil string will contain a valid schema.

Reply:
Thanks both.